isis/src/control/objs/BundleUtilities/IsisBundleObservation.cpp

```diff
@@ -1368,8 +1368,7 @@ QStringList IsisBundleObservation::parameterList() {
    *                calculated (line, sample) coordinate
    */
   double IsisBundleObservation::computeObservationValue(BundleMeasure &measure, double deltaVal) {
-    Camera *measureCamera = measure.camera();
-    return deltaVal / measureCamera->PixelPitch();
+    return deltaVal * 1.4;
```

Comment:
This looks suspiciously like std::sqrt(2.0), which is the distance from the center of a square pixel to any one of its corners. The returned value is in pixel units rather than physical size (mm?) of the detector pixel.

Reply:
Kris is correct that it's a shortcut for sqrt(2). This is porting some old logic over so if 1.4 should be used is a bit beyond this PR. See my post on the issue:

https://github.com/USGS-Astrogeology/ISIS3/issues/4655#issuecomment-944361620

Because we don't actually need to be modeling the physical reality of anything, we just need a mathematically consistent system using 1.4 is totally fine as long as it's consistent. In an ideal world, this bit would be completely obsoleted by actual measure weighting but we haven't implemented a good way to do that yet; so, we're just waiting on a sensor by sensor basis.